sqlx-core/src/net/tls/mod.rs

```diff
@@ -117,8 +121,16 @@ async fn configure_tls_connector(
     accept_invalid_certs: bool,
     accept_invalid_hostnames: bool,
     root_cert_path: Option<&CertificateInput>,
+    client_cert_path: Option<&CertificateInput>,
```

Comment:
Can we fix these warnings? 

If the `native-tls` feature is enabled but not `openssl-native`, we should check if these are not `None` and if so, log a warning that these do nothing without the `openssl-native` feature.